app/assets/stylesheets/custom.css.scss

```diff
@@ -83,4 +83,12 @@ body {
   margin-top: -20px;
 }
 
+.footer {
+  position: relative;
```

Comment:
Properties should be ordered background-color, bottom, height, margin-top, position, width